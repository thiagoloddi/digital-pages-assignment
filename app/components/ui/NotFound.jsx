import React from 'react';
import injectSheet from 'react-jss';

const style = {
	container: {
        fontSize: '5rem',
        textAlign: 'center',
        marginTop: '40vh',
        fontWeight: 'bold',
        color: 'grey'
    }
}

const NotFound = ({ classes }) => {
  return (
    <div className={classes.container}>
      404 NOT FOUND
    </div>
  )
}

export default injectSheet(style)(NotFound);