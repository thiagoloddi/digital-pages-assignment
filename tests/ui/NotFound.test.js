import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../app/components/ui/NotFound';

it('renders correctly', () => {
  const tree = renderer
    .create(<NotFound show={true} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});