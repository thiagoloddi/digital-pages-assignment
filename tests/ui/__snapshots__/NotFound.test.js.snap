// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="NotFound-container-0-1-1"
>
  404 NOT FOUND
</div>
`;
